### ISSUE TITLE
Anchor support in Midi

### ISSUE THREAD

**Issue by TheBoxyBear** [TheNathannator/GuitarGame_ChartFormats#6]
In the section about the tempo track, there is not mention of how anchors are stored. At least format used by Moonscraper, anchors are not supported. Once exported, the chunk only contains information about the time signature and tempo markers. Opening the file in Moonscraper reveals all anchor data is gone.

**Comment by TheNathannator** [comment]
Anchors don't exist in .mid, the data is lost because there's no equivalent for them to export to.

**Comment by TheBoxyBear** [comment]
Yes, the point was that it would be useful to mention it in the docs

**Comment by TheNathannator** [comment]
Ah, that wasn't clear, though I'd figure it'd be kinda implicit that they don't exist if they're not mentioned. MIDI *is* a standard format after all, I've done my best to document all of the relevant info for it.
